Bottom: 9064e9300559d61dc1319424c6e0b5d31ced8a67
Top:    9c4659a19939c5839e0b1a39badde2b8d8b3ce00
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-18 20:51:35 +0000

Refresh of fix-grid-extents-in-3d-view

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 42fe4be..c624d8b 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2983,7 +2983,6 @@ poly_ContourInContour (PLINE * poly, PLINE * inner)
         } while ((pt = pt->next) != &inner->head);
       return 1;
     }
-  }
   return 0;
 }
